xml/System.Data.Common/DbBatch.xml

```diff
@@ -540,9 +633,18 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the wait time (in seconds) before terminating the attempt to execute the batch and generating an error.</summary>
+        <value>The time in seconds to wait for the batch to execute.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ An <xref:System.ArgumentException> is generated if the assigned <xref:System.Data.Common.DbBatch.Timeout%2A> property value is less than 0.  
+  
+ Note to implementers, it is recommended that 0 means no timeout.  
```

Comment:
```suggestion
 Note to implementers: it's recommended that 0 means no timeout.  
```

Reply:
Thanks, fixed this in DbCommand as well. I've also replaced `means` with `mean`:

```
Note to implementers: it's recommended that 0 mean no timeout.
```